src/masternode-sync.cpp

```diff
@@ -508,6 +508,21 @@ void CMasternodeSync::ProcessTick()
     ReleaseNodes(vNodesCopy);
 }
 
+void CMasternodeSync::SendGovernanceSyncRequest(CNode* pnode, const char* strMessage, uint256 nHash)
+{
+    if(pnode->nVersion >= GOVERNANCE_FILTER_PROTO_VERSION) {
+        CBloomFilter filter(GOVERNANCE_FILTER_ELEMENTS, GOVERNANCE_FILTER_FP_RATE, GetRandInt(999999), BLOOM_UPDATE_ALL);
+        if(!filter.IsWithinSizeConstraints()) {
```

Comment:
empty filter should always be within constraints, so after the fix above this shouldn't be needed